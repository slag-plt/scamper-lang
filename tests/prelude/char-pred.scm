(char-alphabetic? #\c)
(char-alphabetic? #\5)
(char-alphabetic? #\ )

(char-numeric? #\c)
(char-numeric? #\5)
(char-numeric? #\ )

(char-whitespace? #\c)
(char-whitespace? #\5)
(char-whitespace? #\ )

(char-upper-case? #\c)
(char-upper-case? #\F)
(char-upper-case? #\newline)

(char-lower-case? #\c)
(char-lower-case? #\F)
(char-lower-case? #\newline)