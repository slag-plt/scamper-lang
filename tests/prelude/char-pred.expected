#t
#f
#f
#f
#t
#f
#f
#f
#t
#f
#t
#f
#t
#f
#f
